ocs_ci/ocs/resources/pvc.py

```diff
@@ -308,6 +308,28 @@ def get_all_pvcs_in_storageclass(storage_class):
     return out
 
 
+def get_all_pvs_in_storageclass(storage_class):
```

Comment:
Also, oc get pv always returns PVs across all namespaces. I don't think the storage class matters.